docroot/modules/custom/mass_entity_usage/mass_entity_usage.module

```diff
@@ -146,3 +146,13 @@ function mass_entity_usage_entity_insert(EntityInterface $entity) {
 function mass_entity_usage_entity_update(EntityInterface $entity) {
   mass_entity_usage_keep_only_current_vid($entity);
 }
+
+/**
+ * Implements hook_queue_info_alter().
+ */
+function mass_entity_usage_queue_info_alter(&$queues) {
+  // Removes entity usage queue from processing when cron runs.
+  if (isset($queues['entity_usage_regenerate_queue'])) {
+    $queues['entity_usage_regenerate_queue']['cron']['time'] = 0;
```

Comment:
```suggestion
    unset($queues['entity_usage_regenerate_queue']['cron']);
```
I didn't test this so please let us know if this works.

Reply:
Right, based on `docroot/core/lib/Drupal/Core/Cron.php:171` is even better to unset instead of setting to 0.
I will test locally anyway. Thanks!

Reply:
Tested locally. That works too.